src/core/ObservableQuery.ts

```diff
@@ -730,12 +731,18 @@ Did you mean to call refetch(variables) instead of refetch({ variables })?`);
 
     const info = pollingInfo || (this.pollingInfo = {} as any);
     info.interval = pollInterval;
+  
+    // The fetchPolicy to be used when polling. Defaults to network-only.
+    let pollFetchPolicy:PollFetchPolicy = 'network-only';
+    if (this.options.fetchPolicy === 'no-cache') {
+      pollFetchPolicy = 'no-cache';
+    }
 
     const maybeFetch = () => {
       if (this.pollingInfo) {
         if (!isNetworkRequestInFlight(this.queryInfo.networkStatus)) {
           this.reobserve({
-            fetchPolicy: "network-only",
+            fetchPolicy: pollFetchPolicy,
```

Comment:
On second thought, since that `pollFetchPolicy` variable is used only once, maybe we could just inline it down here (rendering my suggestion above irrelevant)?
```suggestion
            fetchPolicy: this.options.fetchPolicy === 'no-cache' ? 'no-cache' : 'network-only',
```

Reply:
Even though the variable is only used once, I think I'll still go with your suggestion above so we can easily get use out of the `PollFetchPolicy` type

Reply:
Psyche! Per our discussion going with this instead, removing our type

Reply:
Per our discussion, actually going in favor of this instead and removing the PollFetchpolicy type